java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -1211,4 +1214,43 @@ CompletableFuture<Map<String, Double>> zunionWithScores(
      * }</pre>
      */
     CompletableFuture<Object[][]> zrandmemberWithCountWithScores(String key, long count);
+
+    /**
+     * Returns the cardinality of the intersection of the sorted sets specified by <code>keys</code>.
+     *
+     * @apiNote When in cluster mode, all <code>keys</code> must map to the same hash slot.
+     * @since Redis 7.0 and above
+     * @see <a href="https://redis.io/commands/zintercard/">redis.io</a> for more details.
+     * @param keys The keys of sorted sets to intersect.
+     * @param limit Specifies a maximum number for the intersection cardinality. If limit is set to
+     *     <code>0</code> the range will be unlimited.
+     * @return The number of members in the intersection, or the <code>limit</code> if reached.
+     * @example
+     *     <pre>{@code
+     * Long length1 = client.zintercard(new String[] {"mySortedSet1", "mySortedSet2"}, 5).get();
+     * assert payload1 == 3L;
+     *
+     * Long length2 = client.zintercard(new String[] {"mySortedSet1", "mySortedSet2"}, 0).get();
+     * assert payload2 == 10L;
+     * }</pre>
+     */
+    CompletableFuture<Long> zintercard(String[] keys, long limit);
+
+    /**
+     * Returns the cardinality of the intersection of the sorted sets specified by <code>keys</code>.
+     * If the intersection cardinality reaches <code>limit</code> partway through the computation, the
```

Comment:
I think this should be in the function above as this functions does not have `limit`